packages/webapp/next-seo.ts

```diff
@@ -23,7 +23,7 @@ export const defaultSeo: Partial<NextSeoProps> = {
 export const defaultOpenGraph: Partial<OpenGraph> = {
   images: [
     {
-      url: 'https://daily-now-res.cloudinary.com/image/upload/v1680599225/opengraph/Open_Graph_3.png',
+      url: 'https://res.cloudinary.com/daily-now/image/upload/s--VAY5ToZt--/f_auto/v1722860388/public/daily.dev%20-%20open%20graph',
```

Comment:
should we also add `q_auto`

Reply:
no need in this case. also since these are signed urls you can't easily add them (fyi)

Reply:
I think so, it goes from 404 KiB to 154 KiB with `q_auto`

Reply:
yes but we want to keep the quality of the image. at some point design commented about q_auto messing with the quality

Reply:
Well it's exactly about quality. I'll leave it to what we have atm unless instructed otherwise.

Reply:
404kb seems a bit large for og image, but yeah, if they requested it (can't see thread, it opens nothing) 🙈